Azure-Samples/azure-intelligent-edge-patterns
# Inference Benchmark for GPU using resnet50 (various batch sizes with expected throughput and latency comparision)

## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[ x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[x ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code
*  Deploy and run the module as it is run today - you should see the inference benchmark results sent out along with the other sample results that are being sent today

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

## Other Information
